frontend/customization/config/map.json

```diff
@@ -4,32 +4,32 @@
   "searchMapZoom": 10,
   "mapClassicLayers": [
     {
-      "url": "https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png",
+      "url": "//{s}.tile.opentopomap.org/{z}/{x}/{y}.png",
       "options": {
         "attribution": "Map data: © <a href=\"https://www.openstreetmap.org/copyright\" target=\"_blank\">OpenStreetMap</a> contributors, SRTM | Map style: © <a href=\"https://opentopomap.org\" target=\"_blank\">OpenTopoMap</a> (<a href=\"https://creativecommons.org/licenses/by-sa/3.0/\" target=\"_blank\">CC-BY-SA</a>)",
         "minZoom": 1,
-        "maxZoom": 14
+        "maxZoom": 11
       }
     },
     {
-      "url": "https://wxs.ign.fr/cartes/geoportail/wmts?&REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&STYLE=normal&TILEMATRIXSET=PM&FORMAT=image/png&LAYER=GEOGRAPHICALGRIDSYSTEMS.PLANIGNV2&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}",
+      "url": "//wxs.ign.fr/cartes/geoportail/wmts?&REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&STYLE=normal&TILEMATRIXSET=PM&FORMAT=image/png&LAYER=GEOGRAPHICALGRIDSYSTEMS.PLANIGNV2&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}",
       "options": {
         "attribution": "© IGN",
-        "minZoom": 15,
+        "minZoom": 12,
         "maxZoom": 20
       }
     }
   ],
   "mapSatelliteLayers": [
     {
-      "url": "https://wxs.ign.fr/ortho/geoportail/wmts?&REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&STYLE=normal&TILEMATRIXSET=PM&FORMAT=image/jpeg&LAYER=ORTHOIMAGERY.ORTHOPHOTOS&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}",
+      "url": "//wxs.ign.fr/ortho/geoportail/wmts?&REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&STYLE=normal&TILEMATRIXSET=PM&FORMAT=image/jpeg&LAYER=ORTHOIMAGERY.ORTHOPHOTOS&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}",
       "options": {
         "attribution": "© IGN"
       }
     }
   ],
   "mapOfflineLayer": {
-    "url": "https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png",
+    "url": "//{s}.tile.opentopomap.org/{z}/{x}/{y}.png",
```

Comment:
Maybe in another PR but Opentopomap is less and less reliable, what alternative can we use?

Reply:
Yes, Opentopomap is a small styling project by a German university, which has small servers and no disponibility guaranted.

To be used in good conditions, it would require to host ourself a big Opentopomap server.

I think it shouldn't be used as default basemap.

What we should rather do, is use Openstreetmap basemap as default (``https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png``), even if it doesn't include relief (which is the main interest of Opentopomap, for hiking) or IGN Plan v2 (``https://wxs.ign.fr/cartes/geoportail/wmts?&REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&STYLE=normal&TILEMATRIXSET=PM&FORMAT=image/png&LAYER=GEOGRAPHICALGRIDSYSTEMS.PLANIGNV2&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}``) which is now open without key. But this one is only available for France.

Reply:
I agree, OpenTopoMap is not good as alternative.

As I can say, all other raster tile providers allows use if it is marginal, otherwise the advise to contact them.
https://wiki.openstreetmap.org/wiki/Raster_tile_providers

For example, CycloOSM tile provider
https://operations.osmfoundation.org/policies/tiles/

OSM France provide Humatarian too, in the same conditions
https://www.openstreetmap.fr/fonds-de-carte/

If we use one of these tiles provide in example, Maybe we should advise users to contact provider, for example in credits mentions ?

This could be set in a specific merge request to avoid this one blocked.

Reply:
Ok, to let this discuss to another PR!